wsd/DocumentBroker.hpp

```diff
@@ -491,8 +492,13 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
             _status = status;
         }
 
+        /// Flag close-requested. Cannot be reset.
+        void setCloseRequested() { _closeRequested = true; }
+        bool isCloseRequested() const { return _closeRequested; }
```

Comment:
Shouldn't we actually then have const std::atomic<bool>& as the return value here?

Reply:
That won't be necessary, because these flags change only once (at most) in the lifetime of DocBroker, and we don't repeatedly check the value to see they are changed. Alternatively, we call this function every time we need the latest value, so, again, no need to take a reference: the next call is guaranteed to give us the last value.